### DISCUSSION TITLE
Unable to Deep-Link to Specific Code Blocks in README Files

### DISCUSSION THREAD

**Discussion by 0xZeyad11** [community/community#61830] [Repositories]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

**Bug Description:**

I am encountering an issue related to deep-linking specific code blocks within the README files of my GitHub repository. When attempting to link a particular block of code with specific line numbers, the link redirects to a `not found` page. Notably, this problem seems to be limited to the GitHub website on Desktop browsers.

**Steps to Reproduce:**

1. Visit the README file in the repository: <https://github.com/zedoh/DS-Algo>
2. Try to deep-link to a specific function with line numbers using the following Markdown syntax:
   ```
   [Name of the specific function](/path/to/file#Lstart-line-Lend-line)
   ```
   Replace `path/to/file` with the relative path to the file containing the code.
   Replace `start-line` and `end-line` with the line numbers defining the specific code block.

**Expected Behavior:**

Upon clicking the deep link, it should navigate to the corresponding code block within the README file.

**Actual Behavior:**

Instead of directing to the specific code block, the link redirects to a `not found` page, indicating that the content is inaccessible. It is essential to note that this issue is exclusive to the GitHub website on desktop browsers, as deep-linking works correctly when accessing the README files through the GitHub Android application.

**Additional Information:**

- Repository Link: [https://github.com/zedoh/DS-Algo](https://github.com/zedoh/DS-Algo)
- One of the Affected Folder: [https://github.com/zedoh/DS-Algo/tree/main/DS/Queue](https://github.com/zedoh/DS-Algo/tree/main/DS/Queue)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @zedoh ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
